go.mod

```diff
@@ -26,6 +26,7 @@ replace (
 	// LOOK TO scripts/download FOR THE VERSION OF runc THAT WE ARE BUILDING/SHIPPING
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc93.0.20210414171415-3397a09ee932
 	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.3-0.20210316141917-a8c4a9ee0f6b
+	github.com/rancher/k3s/pkg/data => ./pkg/data
```

Comment:
fixes a nit: if you're on a clean checkout and run `go mod tidy` it pings out to the go checksum db looking for this module. This makes it explicit that we should find it locally, and gives a more meaningful error if it is not.